packages/client/hmi-client/src/components/llm/tera-beaker-input.vue

```diff
@@ -72,18 +74,14 @@ const addCodeCellButton = ref<HTMLElement | null>(null);
 const fixedDivWidth = ref(0);
 
 const submitQuery = () => {
-	EventService.create(
-		EventType.TransformPrompt,
-		useResourcesStore().activeProject?.id,
-		queryString.value
-	);
+	EventService.create(EventType.TransformPrompt, activeProject.value?.id, queryString.value);
```

Comment:
We should remove the active project from the call, and instead call `useProject.activeProject.value.id` in the `EventService.create()` method.

Reply:
I think we should keep it actually as there are some places where this is used where the project might not be the active project